internal/api/operations/delete.go

```diff
@@ -35,6 +35,17 @@ func (op DeleteOperation) Apply(tx db.RWTx) (*db.QueryResult, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	//final include
+	root = tx.Ref(op.ModelID)
+	clauses = handleFindMany(tx, root, op.FindArgs)
+	q = tx.Query(root, clauses...)
+	outs = q.All()
+	if len(outs) == 0 {
+		return nil, nil
+	}
+	out.ToOne = outs[0].ToOne
```

Comment:
can you explain this one?

Reply:
It was to give the original record with the new includes / selects. What we talked about in terms of desired behavior for how things return

Reply:
OK please add a comment explaining it